### DISCUSSION TITLE
Sharing via twitter

### DISCUSSION THREAD

**Discussion by blogdemoi** [FreshRSS/FreshRSS#4516] [Q&A]
Hi,

How can I share an article on twitter?

Thank you.

**Comment by Alkarex** [comment]
1. ![image](https://user-images.githubusercontent.com/1008324/185812279-fa1fa9ac-c40e-40c6-b8ba-0e7f25cd47fd.png)

2. ![image](https://user-images.githubusercontent.com/1008324/185812307-46f3ce1f-5e33-4517-affa-5000906d23ad.png)

**Reply by blogdemoi** [reply]
Thank you.

In my Freshrss install (on Yunohost) I don't have any share button : 

![Aen4Xqq](https://user-images.githubusercontent.com/83965802/185818755-a3ec523e-f9d8-45cc-8fc9-c8d0da09c1fb.png)

I did the share configuration though, as indicated in your screenshot. My Freshhrss version is 1.19.2.

**Reply by Alkarex** [reply]
Try with the *normal view*
![image](https://user-images.githubusercontent.com/1008324/185901442-3413e9b9-19fd-4a39-930f-d61bb47a3837.png)

**Reply by blogdemoi** [reply]
Thank you, but the share button is nowhere. I double checked my settings. Maybe it is an issue with the Yunohost packaging of the application, I opened an issue there: https://github.com/YunoHost-Apps/freshrss_ynh/issues/143 , wait and see.

**Reply by Alkarex** [reply]
Could you please post a screenshot in the *normal* view (and not in the *reader* view)?

**Reply by blogdemoi** [reply]
Here are the screenshots:

Normal view:
![2022-Monday-August-18-38-29](https://user-images.githubusercontent.com/83965802/185974114-5dab6d29-5293-4465-a8d5-d9bef1742c01.png)

Reader view:
![2022-Monday-August-18-39-24](https://user-images.githubusercontent.com/83965802/185974284-1ef0bd16-6169-45a0-918e-8df922e3b0d4.png)

My settings:
![2022-Monday-August-18-39-49](https://user-images.githubusercontent.com/83965802/185974338-c00d0c17-57b2-4853-8fa2-79f672ec8ef4.png)

![2022-Monday-August-18-40-16](https://user-images.githubusercontent.com/83965802/185974366-5df5a2bc-95ce-4434-b115-b7432fe4c520.png)

**Reply by Alkarex** [reply]
What if you open an article in normal view?

**Reply by blogdemoi** [reply]
> What if you open an article in normal view?

Yes, thank you, in that case the share icon is here:
![2022-Monday-August-19-00-44](https://user-images.githubusercontent.com/83965802/185978146-032b066e-d2d2-4abc-8f68-d4e5e165cce3.png)

And I can select an e-mail client:
![2022-Monday-August-19-05-35](https://user-images.githubusercontent.com/83965802/185978568-c314ad1d-a720-486c-8cf3-151d5cb28718.png)

However when I click on twitter it redirects to the article web page. I guess it is because the twitter sharing option is not configured. In the sharing settings of Freshrss it is written: "not available":
![2022-Monday-August-19-04-43](https://user-images.githubusercontent.com/83965802/185979152-ad32fdb3-846b-46fc-bf44-bbb47223122d.png)

How can I configure it? Or make it available?

**Reply by Frenzie** [reply]
That's just some bad UI. Some options take a URL there, but Twitter is Twitter.
